app/packages/app/src/recoil/atoms.ts

```diff
@@ -68,6 +68,11 @@ export const fullscreen = atom<boolean>({
   default: false,
 });
 
+export const showLabels = atom<boolean>({
```

Comment:
I'm not sure if this is needed. Is there another atom / place that allows mutating the options? I'm just following the fullscreen toggle implementation.